src/trove/lists.scrbl

```diff
@@ -1603,6 +1548,20 @@ end
 }
 
 
+@list-method["join-str-last"]
+Combines the values of the current @pyret{List} by converting them to strings
+with @pyret{tostring} and joining them with the given separator @pyret{sep}.
+If the list has more than one element, the function will use @pyret{last-sep}
+to join the last element instead of the regular @pyret{sep}.
+@examples{
+check:
+  [list: 1, 2, 3].join-str-last("; ", "$ ") is "1; 2$ 3"
```

Comment:
These examples are deliberately obscure.  Probably a better example would be to produce "1, 2, 3 and 4", or something a bit more human-relevant.

Reply:
You should also have an example of `[list: 1].join-str("sep", "last") is "1"`, similar to your last example below, to drive home the degenerate cases when one or both separators aren't needed.

Reply:
Noted. Will fix.

Reply:
Fixed